src/components/notifications/Notifications.tsx

```diff
@@ -142,17 +141,6 @@ const Notifications = ({
     markAsViewed()
   }, [initialPosition, close, markAsViewed, backgroundAnim, translationAnim])
 
-  useAppState(
```

Comment:
@raymondjacobson After messing with entering/exiting background with the notifications open, I couldn't find a case were this was needed. So removing it for now, let's keep and eye on this during testing though